src/app/containers/MediaPlayer/fixtureData.jsx

```diff
@@ -279,3 +288,97 @@ export const missingVpidFixture = [
     position: [4, 2],
   },
 ];
+
+const defaultToggles = {
+  local: {
+    mediaPlayer: {
+      enabled: true,
+    },
+  },
+  test: {
+    mediaPlayer: {
+      enabled: true,
+    },
+  },
+  live: {
+    mediaPlayer: {
+      enabled: false,
+    },
+  },
+};
+
+const toggleStateOff = {
+  local: {
+    mediaPlayer: {
+      enabled: false,
+    },
+  },
+};
+
+const GenerateFixtureData = ({
+  platform,
+  toggleState,
+  blocks,
+  placeholder,
+}) => (
+  <RequestContextProvider
+    isAmp={platform === 'amp'}
+    service="news"
+    statusCode={200}
+    platform={platform}
+    id="foo"
+    pageType="article"
+    pathname="/pathname"
+  >
+    <ServiceContextProvider service="news">
+      <ToggleContext.Provider
+        value={{ toggleState, toggleDispatch: jest.fn() }}
+      >
+        <MediaPlayerContainer
+          blocks={Array.isArray(blocks) ? blocks : [blocks]}
```

Comment:
When might `blocks` be a single block?

Reply:
Single block: https://github.com/bbc/simorgh/blob/ce04f0f1262c3080029a033832ef2b028c70f7e0/src/app/containers/MediaPlayer/fixtureData.jsx#L50
Multiple blocks: https://github.com/bbc/simorgh/blob/ce04f0f1262c3080029a033832ef2b028c70f7e0/src/app/containers/MediaPlayer/fixtureData.jsx#L244

Reply:
I'd leave this to the caller to minimise confusion over types.